app/controllers/items_controller.rb

```diff
@@ -1,9 +1,9 @@
 class ItemsController < ApplicationController
   def index
-    if params[:category_name]
-      @items = Category.find_by(name: params[:category_name]).items
-    else
+    if params[:category_name] == 'Shop All' || params[:category_name].nil?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.